app/src/org/commcare/utils/SessionActivityRegistration.java

```diff
@@ -41,8 +41,6 @@ public void onReceive(Context context, Intent intent) {
      * methods of activities that are session sensitive.
      */
     public static boolean handleOrListenForSessionExpiration(AppCompatActivity activity) {
```

Comment:
can we rename it to just `handleSessionExpiration` since it doesn't register the receiver anymore.